source/jormungandr/tests/departure_board_tests.py

```diff
@@ -1505,7 +1505,7 @@ class TestFirstLastDatetimeWithNegativeTimezone(AbstractTestFixture):
     """
 
     def test_first_last_datetime_same_day(self):
-        """              20170102
+        """20170102
```

Comment:
I'd say the extra spaces here are needed to align the text with these ASCII arts (see the entire file for similar correction).